<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>693</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>693</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Senai Excel Management - Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/logo-senai.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_colunas">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="load_path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 4px;
border: none;</string>
   </property>
   <property name="text">
    <string>Example: 'C:/User/Documents/file.csv...'</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="load_excel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
color: black;
border:none;
border-radius: 4px
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
   </property>
   <property name="text">
    <string>CARREGAR ARQUIVO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_buscar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
color: black;
border:none;
border-radius: 4px
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
   </property>
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_rows">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CAMINHO DO ARQUIVO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>COLUNA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LINHA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_retorno">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>[...]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_path_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RETORNO DA PLANILHA:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>671</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>951</width>
     <height>461</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>cb_colunas</zorder>
  <zorder>load_path</zorder>
  <zorder>load_excel</zorder>
  <zorder>botao_buscar</zorder>
  <zorder>cb_rows</zorder>
  <zorder>label_path</zorder>
  <zorder>label_path_2</zorder>
  <zorder>label_path_3</zorder>
  <zorder>lbl_retorno</zorder>
  <zorder>label_path_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
